pkg/kyverno/test/command.go

```diff
@@ -34,9 +34,9 @@ import (
 
 // Command returns version command
 func Command() *cobra.Command {
-
-	var valuesFile string
-	return &cobra.Command{
+	var cmd *cobra.Command
+	var valuesFile, fileName string
+	cmd = &cobra.Command{
 		Use:   "test",
 		Short: "Shows current test of kyverno",
```

Comment:
Use:   "test" should be Use:   "test <directory>",

change "Shows current test of kyverno" to "run tests from directory"